envsec/pkg/stores/ssmstore/ssmstore.go

```diff
@@ -30,14 +32,18 @@ func newSSMStore(ctx context.Context, config *SSMConfig) (*SSMStore, error) {
 	return store, nil
 }
 
-func (s *SSMStore) List(ctx context.Context, envID EnvID) ([]EnvVar, error) {
+func (s *SSMStore) Identify(context.Context, *envsec.Envsec, *session.Token) {
+	// TODO: implement
```

Comment:
maybe add a `panic("TODO: implement SSMStore.Identify")` ?

Reply:
I don't really want it to panic since it does identity in a different way (via standard AWS credential chain). If we make it panic here it will break.